src/Serilog.Framework.Logging/SerilogLoggerProvider.cs

```diff
@@ -57,7 +57,7 @@ public void Enrich(LogEvent logEvent, ILogEventPropertyFactory propertyFactory)
             }
         }
         
-#if DNXCORE50
+#if NETCORE50 || DOTNET5_4
```

Comment:
Because the Netstandard/Dotnet targets are going to become the standard, is it possible or would it make more sense for these code blocks to be inverted? i.e. This code is the default, and we can `#if` any targets that don't support the standard code?